home02.py

```diff
@@ -0,0 +1,9 @@
+a = input("Write something: ")
+if (a.isdigit()) and (int(a) % 2 == 0):
```

Comment:
There is no need to use parentheses around the condition. You can use it like this: `if a.isdigit()) and (int(a) % 2 == 0:`